auth-jwt-service/src/main/java/io/stargate/auth/jwt/PlainTextJwtTokenSaslNegotiator.java

```diff
@@ -21,20 +21,24 @@
 import io.stargate.auth.Credentials;
 import io.stargate.auth.PlainTextTokenSaslNegotiator;
 import io.stargate.db.Authenticator.SaslNegotiator;
+import java.util.Map;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class PlainTextJwtTokenSaslNegotiator extends PlainTextTokenSaslNegotiator {
 
   private static final Logger logger =
       LoggerFactory.getLogger(PlainTextJwtTokenSaslNegotiator.class);
+  private final Map<String, String> headers;
 
   public PlainTextJwtTokenSaslNegotiator(
       AuthenticationService authentication,
       SaslNegotiator wrapped,
       String tokenUsername,
-      int tokenMaxLength) {
+      int tokenMaxLength,
+      Map<String, String> headers) {
```

Comment:
Not a big deal, but this is weird passing  "HTTP headers" around in the CQL code. Because of this I lean toward calling it "properties" instead of "headers".

Reply:
Nothing in the name imposes that those are `http headers`. There are just `headers`. I can change it to properties if you prefer. But in that case, the name should be changed only in the `cql` related code, right? The non-CQL code should still name this parameter as `headers`?

Reply:
+1 for anything cql related being referred to as properties.

Reply:
I am not sure about boundaries where they should be called `headers` and where `properties`.
For example, `PlainTextJwtTokenSaslNegotiator` is not in the `CQL` module (or package) so I am not sure if this can be used only in the cql context.

Reply:
Yes, that is only used for CQL